server/service/individualServices/SoftwareUpgrade.js

```diff
@@ -217,7 +220,7 @@ async function PromptForBequeathingDataCausesTransferOfListOfAlreadyGrantedTypeA
                      ************************************************************************************/
                     let requestBody = {};
                     requestBody.applicationName = applicationName;
```

Comment:
could you please fetch the information from application-data.json file instead of application profile as application profile is no more valid from v2.0.0

Reply:
done